python/lsst/ip/diffim/getTemplate.py

```diff
@@ -549,13 +537,14 @@ def __init__(self, *args, **kwargs):
     def runQuantum(self, butlerQC, inputRefs, outputRefs):
         # Read in all inputs.
         inputs = butlerQC.get(inputRefs)
-        inputs['coaddExposures'] = self.getOverlappingExposures(inputs)
-        # SkyMap only needed for filtering without
-        inputs.pop('skyMap')
-        outputs = self.run(**inputs)
+        coaddExposuresRefs, patchList = self.getOverlappingExposures(inputs['coaddExposures'],
```

Comment:
You free for paircoding tomorrow? I have an idea:

- Have `getOverlappingExposures` take the deferredDatasetHandles, construct the DCR models, and return actual `exposures` (not deferredDatasetHandles).  (6 patches is an acceptable number to have in memory)
- Keep `run` responsible for just the warping + `CoaddPsf` construction part.